component.css

```diff
@@ -0,0 +1,29 @@
+@import "/Components/Alert/alert.css";
+@import "/Components/Badge/badge.css";
+
+
+:root{
+    /* Main Colors */
+    --PRIMARY-COLOR:#701a75;
+    --SECONDARY-COLOR:#a855f7;
+    --BACKGROUND-COLOR:YELLOW;
+    --TEXT-COLOR:#e9d5ff;
+
+    /* Component Colors */
+    --DANGER-COLOR:#dc2626;
+    --SUCESS-COLOR:#65a30d;
+    --INFO-COLOR:#2563eb;
+    --WARNING-COLOR:#ca8a04;
+
```

Comment:
Nit: Space

Reply:
Reduced the changes. Thankyou for reviewing.